receiver/googlecloudspannerreceiver/README.md

```diff
@@ -52,9 +53,10 @@ receivers:
 
 Brief description of configuration properties:
 - **googlecloudspanner** - name of the Cloud Spanner Receiver related section in OpenTelemetry collector configuration file
-- **collection_interval** - this receiver runs on an interval. Each time it runs, it queries Google Cloud Spanner, creates metrics and sends them to the next consumer(60 seconds by default)
+- **collection_interval** - this receiver runs on an interval. Each time it runs, it queries Google Cloud Spanner, creates metrics and sends them to the next consumer(60 seconds by default). **It is not recommended change the default value fo collection interval, since new values for metrics in Spanner database appear with 1 minute interval.**
```

Comment:
```suggestion
- **collection_interval** - this receiver runs periodically. Each time it runs, it queries Google Cloud Spanner, creates metrics, and sends them to the next consumer (default: 1 minute). **It is not recommended to change the default value of collection interval, since new values for metrics in the Spanner database appear only once a minute.**
```

Reply:
Done.